client/landing/subscriptions/tab-views/Site.tsx

```diff
@@ -0,0 +1,30 @@
+import { useTranslate } from 'i18n-calypso';
+import SiteRow from './SiteRow';
+import { SiteType } from './site-types';
+
+type SiteProps = {
+	sites: SiteType[];
+};
+
+export default function Site( { sites }: SiteProps ) {
```

Comment:
Hmm, `Site` does not sound right to me as a name for the component. The component is responsible for displaying a list of sites. What do you think yourself, @JuanLucha? 🤔

Reply:
True, `Sites` or `SiteList` sounds better. In fact, having the child named `SiteRow`, `SiteList` sounds good to me.